src/test/regress/greenplum_schedule

```diff
@@ -296,4 +296,7 @@ test: create_extension_fail
 # check the row/block statistics is gathered for auxiliary relations of AO table
 test: aux_ao_rels_stat
 
+#check gp_dist_wait_status not failing within a transaction
```

Comment:
If you want to explain just one test (not the file), it's better to add a comment near the test. It's ok not to have a comment in the schedule file if test name is clear.

Reply:
Done